index.html

```diff
@@ -255,6 +255,62 @@
 
 ---
 
+# Continuous Analysis
+
+The Status Quo Doesn't Work:
+.task[
```

Comment:
I'm using .task for something that I want them to do. It's not entirely clear to me what they are supposed to do here. Does this need to be a .task?